app/models/affiliate.rb

```diff
@@ -0,0 +1,8 @@
+class Affiliate < ApplicationRecord
+
+    has_many :coordinators
+    has_many: tutors
+
+    has_many :matches
```

Comment:
I was thinking students' affiliates should be the same as their tutors'. For making sure their affiliates are synced with their tutors', affiliates should have students through matches. But I missed the unmatched students case. You are right! 

We should discuss tomorrow about how to make sure students' affiliates are synced with their tutors'.